source/manual/business-readiness-finder.html.md

```diff
@@ -0,0 +1,67 @@
+
+For a list of all of the YAML files that drive the finder and its related pages, see: [Publish changes to the business readiness finder](/manual/business-readiness-publish-changes.html).
+
+## Overview
+
+The full journey for the business readiness finder is made up of the following content items:
+
+1. The start page: [`/business-uk-leaving-eu`](https://www.gov.uk/business-uk-leaving-eu)
+2. The Q&A: [`/prepare-business-uk-leaving-eu`](https://www.gov.uk/prepare-business-uk-leaving-eu)
+3. The finder: [`/find-eu-exit-guidance-business`](https://www.gov.uk/find-eu-exit-guidance-business)
+4. The email signup page: [`/find-eu-exit-guidance-business/email-signup`](https://www.gov.uk/find-eu-exit-guidance-business/email-signup)
+
+And editors from other departments can request changes to the finder via the [Support](https://github.com/alphagov/support) app, in which we have created a [support form](https://support.integration.publishing.service.gov.uk/eu_exit_business_readiness_request/new).
+
+### Start page
+
+A normal page which can be updated in Mainstream Publisher as you would any other start page. (No modifications).
+
+### Q&A
+
+The Q&A is [documented in finder-frontend](https://github.com/alphagov/finder-frontend/blob/master/docs/q-and-a.md).
+
+### Finder
+
+The business finder is driven by 'links' (rather than 'tags', though we often refer to this process as 'tagging'). These links come from the Publishing API.
+
+Any piece of content can be assigned any link, each of which is independent; there is no concept of a link hierarchy or grouping. In order to group links into separate facets, such as "Sector / Business Area", we had to [define the groupings](https://github.com/alphagov/content-tagger/blob/master/lib/data/find-eu-exit-guidance-business.yml) in the content tagger. Any changes to this YAML file affect both the facets in the finder and the options in the Q&A.
+
+We added the [ability to 'pin' content items](https://github.com/alphagov/content-tagger/pull/884) (via the content-tagger) to the top of finder results. This feature is not currently in use and could be removed. The corresponding [pinning code in the support app](https://github.com/alphagov/support/pull/574) should also be removed.
+
+We also added the ability to [sort by topic](https://github.com/alphagov/finder-frontend/pull/856). This is only applicable to the business readiness finder, but is not a candidate for deletion, as it's been proven to be a useful feature.
+
+The finder uses the search-api to find and filter results, using the standard 'OR logic' between facets, where the more options you select, the more results you see. There is [legacy 'AND logic' code](https://github.com/alphagov/search-api/pull/1501 which is no longer being used, and should be removed.
```

Comment:
I think this should be re-worded. "OR logic" is not standard behaviour for a finder, though it is the default way that filtering by links works. The same goes for "legacy AND logic". I know that you mean, the code added to support "ANDs between links, but that's not clear from the wording here.

Reply:
I've pushed up a change, could you check if it's correct now and help me answer the question here: https://github.com/alphagov/govuk-developer-docs/pull/1804/files#r295292352 ?